src/strings.php

```diff
@@ -0,0 +1,301 @@
+__('WordPress Integrity (%%SUCURI.Integrity.ListCount%%)', 'sucuri-scanner');
+__('Status', 'sucuri-scanner');
+__('File Size', 'sucuri-scanner');
+__('Modified At', 'sucuri-scanner');
+__('File Path', 'sucuri-scanner');
+__('Marking one or more files as fixed will force the plugin to ignore them during the next scan, very useful when you find false positives. Additionally you can restore the original content of the core files that appear as modified or deleted, this will tell the plugin to download a copy of the original files from the official WordPress repository. Deleting a file is an irreversible action, be careful.', 'sucuri-scanner');
+
+// integrity.html.tpl
+__('WordPress Integrity', 'sucuri-scanner');
+__('We inspect your WordPress installation and look for modifications on the core files as provided by WordPress.org. Files located in the root directory, wp-admin and wp-includes will be compared against the files distributed with v%%SUCURI.WordPressVersion%%; all files with inconsistencies will be listed here. Any changes might indicate a hack.', 'sucuri-scanner');
+__('Loading...', 'sucuri-scanner');
+
+// lastlogins.html.tpl
+__('All Users', 'sucuri-scanner');
+__('Admins', 'sucuri-scanner');
+__('Logged-in Users', 'sucuri-scanner');
+__('Failed logins', 'sucuri-scanner');
+
+// register-site.html.tpl
+__('An API key is required to activate some additional tools available in this plugin. The keys are free and you can virtually generate an unlimited number of them as long as the domain name and email address are unique. The key is used to authenticate the HTTP requests sent by the plugin to an API service managed by Sucuri Inc.', 'sucuri-scanner');
+__('If you experience issues generating the API key you can request one by sending the domain name and email address that you want to use to <a href="mailto:info@sucuri.net">info@sucuri.net</a>. Note that generating a key for a website that is not facing the Internet is not possible because the API service needs to validate that the domain name exists.', 'sucuri-scanner');
+__('Website:', 'sucuri-scanner');
+__('E-mail:', 'sucuri-scanner');
+__('DNS Lookups', 'sucuri-scanner');
+__('Check the box if your website is behind a known firewall service, this guarantees that the IP address of your visitors will be detected correctly for the security logs. You can change this later from the settings.', 'sucuri-scanner');
+__('Enable DNS Lookups On Startup', 'sucuri-scanner');
+__('I agree to the <a target="_blank" href="https://sucuri.net/terms">Terms of Service</a>.', 'sucuri-scanner');
+__('I have read and understand the <a target="_blank" href="https://sucuri.net/privacy">Privacy Policy</a>.', 'sucuri-scanner');
+__('Submit', 'sucuri-scanner');
+
+// settings-alerts-bruteforce.html.tpl
+__('Password Guessing Brute Force Attacks', 'sucuri-scanner');
```

Comment:
Many of the strings in `strings.php` are not included in `sucuri-scanner.pot`, such as 'Password Guessing Brute Force Attacks'

Reply:
Needed a POT update, done in https://github.com/Sucuri/sucuri-wordpress-plugin/pull/72/commits/ab9d46deb4d416f4aa0565fd6fd8a51c4b15f1a5